config/migrations/2023/20231112161533-update-phone-numbers.sparql

```diff
@@ -0,0 +1,86 @@
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+  ?s <http://schema.org/telephone> ?o .
+  }
+}
+INSERT {
+ GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+   ?s <http://schema.org/telephone> ?newPhoneNumber .
+ }
+}
+WHERE {
+  GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+    ?s <http://schema.org/telephone> ?o .
+  }
+  
+  FILTER (SUBSTR(?o, 1, 1) != "+" AND STRLEN(STR(?o)) > 4 and !regex(?o, ".*[a-zA-Z].*", "i"))
+  BIND(REPLACE(STR(?o),"[. -/()]","") AS ?removedSpaces) .
+  BIND (CONCAT('tel:+32', SUBSTR(?removedSpaces, 2)) as ?newPhoneNumber)
+}
+
+;
+
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+  ?s <http://schema.org/telephone> ?o .
+  }
+}
+INSERT {
+ GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+   ?s <http://schema.org/telephone> ?newPhoneNumber .
+ }
+}
+WHERE {
+  GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+    ?s <http://schema.org/telephone> ?o .
+  }
+  
+  FILTER (SUBSTR(?o, 1, 1) = "+" and !regex(?o, ".*[a-zA-Z].*", "i"))
+  BIND(REPLACE(STR(?o),"[. -/()]","") AS ?removedSpaces) .
+  BIND (CONCAT('tel:', ?removedSpaces) as ?newPhoneNumber)
+}
```

Comment:
These two migrations result in different outcomes. The first one results in phone number literals starting with `tel:+32`, whereas the second one results in literals starting with `tel:32`. Note the missing **+** in the second case.